src/server.c

```diff
@@ -1126,6 +1110,40 @@ void clientsCron(void) {
     }
 }
 
+/* A periodic timer that performs client maintenance.
+ * This cron task follows the following rules:
+ *  - All clients need to be checked (at least) once per second
```

Comment:
i thinks it's unnecessary, if we have 1 million clients, check all of them in one second is very expensive.